util/test/test.hs

```diff
@@ -1,9 +0,0 @@
-import           Universum
```

Comment:
This one is still suspicious.

Reply:
Should be gone once @avieth's changes have been merged into develop.

Reply:
#2861 has been merged into develop.

Reply:
Right, but this branch is based on @avieth avieth/lts112, so I'd need to rebase this branch onto avieth/lts112 once that one is rebased onto develop. Or rebase this one, once avieth/lts112 is marged into develop.

Reply:
Rebasing avieth/lts112 on develop now. Will ping you @angerman when it's done and pushed.